WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -46,6 +46,8 @@
 
 import de.greenrobot.event.EventBus;
 
+import static android.app.Activity.RESULT_OK;
```

Comment:
I think that we could follow the rest of the codebase and just use `Activity.RESULT_OK` where needed and import `Activity` instead of a static import of the constant. WDYT?